src/inputmethodkit.cs

```diff
@@ -12,21 +12,15 @@
 using System;
 
 using ObjCRuntime;
-using Foundation;
+
 using AppKit;
+using Foundation;
+using CoreGraphics;
 
 namespace InputMethodKit {
 
-	partial interface IMKGlobal {
-		[Field ("kIMKCommandMenuItemName")]
-		NSString CommandMenuItemName { get; }
-
-		[Field ("kIMKCommandClientName")]
-		NSString CommandClientName { get; }
-	}
-
-	[Model]
-	partial interface IMKServerProxy {
+	[Protocol]
```

Comment:
Why does this change from `Model` to `Protocol` with the change from `partial interface` to `interface`?

Reply:
Those two changes are not related to each other.